src/pages/My.tsx

```diff
@@ -0,0 +1,32 @@
+import Layout from '../components/Layout';
+import NavBar from '../components/NavBar';
+import { ThemeProvider } from '@emotion/react';
+import GlobalStyle from '../styles/GlobalStyle';
+import theme from '../styles/theme';
+import styled from '@emotion/styled';
+import { useNavigate } from 'react-router-dom';
+
+function My() {
+  const userId = sessionStorage.getItem('userId')?.split('@')[0] ?? '';
+  const userEmail = sessionStorage.getItem('userId') ?? '';
+  const navigate = useNavigate();
+
+  function logOut() {
+    sessionStorage.removeItem('userId');
```

Comment:
이 로그아웃 로직도 useUser에서!

```suggestion
    // const { logout } = useUser();
    
    logout();
```